app/controllers/users.js

```diff
@@ -9,3 +10,11 @@ module.exports.signUp = (req, res, next) =>
       res.status(201).send({ firstName: req.body.firstName });
     })
     .catch(next);
+
+module.exports.signIn = (req, res, next) =>
+  credentialsHelper
+    .signIn(req.body)
+    .then(token => {
+      res.status(200).send({ response: token });
+    })
+    .catch(next);
```

Comment:
All endpoints should have at leats one log detailing what it is doing and showing the param that identifies the main resource. 

Also, something I've just realized (and talked with a TL so it can be added to future trainings):
We have a logger middleware that automatically logs the body of the request (it is deactivated while testing, but if you run the server and call an endpoint you can see it).
Said logs are dangerous because they show sensitive information (passwords in this case). Please configure said middleware to censor the bodies of the signUp and signIn enpoints, as shown here:
https://www.npmjs.com/package/express-wolox-logger#obfuscating-body-of-specific-request

Reply:
You are missing a log in the controller still:
All endpoints should have at leats one log detailing what it is doing and showing the value of the param that identifies the main resource, in this case the email.